packages/core/src/client/sitecore-client.ts

```diff
@@ -404,8 +404,11 @@ export class SitecoreClient implements BaseSitecoreClient {
       return null;
     }
     // If we're in Pages preview (editing) mode, prefetch the editing data
-    const { site, itemId, language, version, variantIds, layoutKind, mode } =
-      previewData as EditingPreviewData;
+    const { site, itemId, language, version, layoutKind, mode } = previewData as EditingPreviewData;
+
+    const variantIds = Array.isArray(previewData.variantIds)
+      ? previewData.variantIds
+      : previewData.variantIds.split(',');
```

Comment:
There's no preview data for nextjs draft mode (used in app router), so preview data is passed via query string. This required an adjustment to how variantIds are handled